resources/swapi/character.yaml

```diff
@@ -0,0 +1,110 @@
+"$schema": https://json-schema.org/draft/2020-12/schema
+"$id": character
+title: Character
+type: object
+required:
+- gid
+- id
+- label
+uniqueKeys:
+- - gid
+- - label
+  - id
+links:
+- rel: homeworld
+  href: homeworld/{id}
```

Comment:
should this be planet/{id}

Reply:
If I understand it right, rel is the name of the relationship so I think it should be homeworld? However, href should be planet/{id}